charts/lnd/charts/loop/Chart.yaml

```diff
@@ -15,9 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.3
+version: 0.1.4
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 0.11.1
+# It is recommended to use it with quotes.
```

Comment:
why is that?

Reply:
Not sure. If you do `helm create xyz`, the Chart.yaml will be generated with this comment i.e. it's not a custom comment